src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/attributes/relationship.svelte

```diff
@@ -0,0 +1,221 @@
+        { value: 'cascade', label: 'Cascade' },
+        { value: 'restrict', label: 'Restrict' }
+    ];
+
+    // Variables
+    let search: string = null;
+    let collectionList: Models.CollectionList;
+    let way = 'one';
+
+    // Lifecycle hooks
+    async function getCollections(search: string = null) {
+        if (search) {
+            const collections = await sdk.forProject.databases.listCollections(
+                databaseId,
+                [Query.orderDesc('$createdAt')],
+                search
+            );
+            return collections;
+        } else {
+            const collections = await sdk.forProject.databases.listCollections(databaseId);
+            return collections;
+        }
+    }
+
+    onMount(async () => {
+        collectionList = await getCollections();
+
+        if (editing) {
+            way = data.twoWay ? 'two' : 'one';
+            search = data.relatedCollection;
+        }
+    });
+
+    // Reactive statements
+
+    $: getCollections(search).then((res) => (collectionList = res));
+    $: collections = collectionList?.collections?.filter((n) => n.$id !== $collection.$id) ?? [];
+
+    $: if (data?.relatedCollection && !data.key) {
+        const collection = collectionList.collections.find((n) => n.$id === data.relatedCollection);
+        data.key = camelize(collection.name);
+    }
+    $: if (way === 'two' && !data.twoWayKey) {
+        data.twoWayKey = camelize($collection.name);
+    }
+
+    $: if (way === 'two') {
+        data.twoWay = true;
+    } else {
+        data.twoWay = false;
+    }
+</script>
+
+<li>
+    <ul
+        class="grid-box"
+        style="--p-grid-item-size:16em; --p-grid-item-size-small-screens:16rem; --grid-gap: 0.5rem;">
+        <li>
+            <LabelCard name="relationship" bind:group={way} value="one" icon="arrow-sm-right">
+                <svelte:fragment slot="title">One-way relationship</svelte:fragment>
+                One Relation attribute within this collection
+            </LabelCard>
+        </li>
+        <li>
+            <LabelCard name="relationship" bind:group={way} value="two" icon="switch-horizontal">
+                <svelte:fragment slot="title">Two-way relationship</svelte:fragment>
+                One Relation attribute within this collection and another within the related collection
+            </LabelCard>
+        </li>
+    </ul>
+</li>
+
+<InputSelectSearch
+    id="related"
+    label="Related Collection"
+    name="collections"
+    bind:search
+    bind:value={data.relatedCollection}
+    required
+    placeholder="Select a collection"
+    disabled={editing}
+    options={collections?.map((n) => ({ value: n.$id, label: n.name })) ?? []} />
+
+{#if data?.relatedCollection}
+    <div>
+        <InputText
+            id="key"
+            label="Attribute Key"
+            placeholder="Enter Key"
+            bind:value={data.key}
+            autofocus
+            required />
```

Comment:
Am I supposed to be able to update the key? It doesn't look like I can for string attributes.